flink-clients/src/main/java/org/apache/flink/client/program/ClusterClient.java

```diff
@@ -163,23 +160,6 @@ public void shutdown() throws Exception {
 	//  Configuration
 	// ------------------------------------------------------------------------
 
-	/**
-	 * Configures whether the client should print progress updates during the execution to {@code System.out}.
-	 * All updates are logged via the SLF4J loggers regardless of this setting.
-	 *
-	 * @param print True to print updates to standard out during execution, false to not print them.
-	 */
-	public void setPrintStatusDuringExecution(boolean print) {
-		this.printStatusDuringExecution = print;
-	}
```

Comment:
Could we deprecate this method and just log a deprecation warning? We could even remove the printing logic, so this would just be a placeholder not to break the @Public contract.